01-numpy.md

```diff
@@ -403,6 +403,18 @@ minimum inflammation: 0.0
 standard deviation: 4.61383319712
 ~~~
 
+> ## Mystery methods in IPython {.callout}
+>
+> How did we know what methods data has and how to use them? When you are working
+> on your own data it might be something different to a numpy object: how will
+> you know what methods you can use then? If you are working in the IPython/Jupyter 
+> Notebook there is an easy way to find out. If you type the name of your object
+> with a full-stop then you can use tab completion (e.g. type `data.` and then press tab)
+> to see a list of all methods that you can use on that object. After selecting one you
+> can also add a question mark (e.g. `data.cumprod?`) and IPython will return an 
+> explanation of the method!
```

Comment:
Specifically, the explanation is the output of `help(data.cumprod)`. Maybe it would be good to mention that as it would tie in with the section on writing docstrings.